src/pybind/mgr/CMakeLists.txt

```
@@ -22,4 +22,7 @@ install(DIRECTORY
   REGEX "rook/rook-client-python.*" EXCLUDE
   REGEX "osd_perf_query/.*" EXCLUDE
   REGEX "tox.ini" EXCLUDE
-  REGEX "requirements.txt" EXCLUDE)
+  REGEX "requirements.txt" EXCLUDE
+  REGEX "node_modules" EXCLUDE
+  REGEX ".editorconfig" EXCLUDE
+  REGEX "cypress.*" EXCLUDE)
```

Not sure if dot files/dirs are copied by cmake install (@tchaikov can you please confirm?), but it might make sense to exclude all of them (e.g.: `.tox`, `.cache`, ...)?
```suggestion
  REGEX "\..+" EXCLUDE
  REGEX "requirements.*\.txt" EXCLUDE
  REGEX "constraints.*\.txt" EXCLUDE  
  REGEX "node_modules" EXCLUDE
  REGEX "\.editorconfig" EXCLUDE
  REGEX "cypress.*" EXCLUDE)
```
You can get inspiration from https://github.com/ceph/ceph/pull/31882/files#diff-97c00666af83ea265ed998e431e56b5c827b490bdaf6ba4e0c196697ccfe1478R24-R28: 
Following paths should also be excluded:
* `wheelhouse`
* `tests`
* Everything under `.../mgr/dashboard/frontend` except `.../dashboard/frontend/dist` (e.g.: `coverage`, `node_modules`, `cypress`, `src`...). I remember looking for a way to exclude everything except a dir and I couldn't find any :/